Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -624,6 +624,31 @@ class SettingsTableViewController: UITableViewController {
         }
     }
 
+    // MARK: Bluetooth Keyboard Shortcuts
+
+    override func canBecomeFirstResponder() -> Bool {
+        return true
+    }
+
+    override var keyCommands: [AnyObject]? {
+        get {
+            return [
+                UIKeyCommand(input: "t", modifierFlags: .Command, action: "openNewTab"), // CMD+T
+                UIKeyCommand(input: "b", modifierFlags: .Command, action: "openNewTab") // CMD+N
```

Comment:
I don't think we want to support these shortcuts here -- the settings page is not part of the browsing context.